src/angular/styles/_typography.scss

```diff
@@ -2499,7 +2499,7 @@ $menuItemPaddingTopBottom: 4;
     }
   }
 
-  .sbb-label {
+  .sbb-label:not(.sbb-autocomplete-hint) {
```

Comment:
I'm uncertain about using `:not` here, but also not opposed to it. Should maybe the `.sbb-label` styles also be applied to the `.sbb-autocomplete-hint`?
@jeripeierSBB, @mhaertwig Opinions?

Reply:
sbb-autocomplete-hint has the sbb-label styles applied, so here I think it's to exclude the padding. I'd suggest to just overwrite padding in sbb-autocomplete-hint if possible